paper.md

```diff
@@ -0,0 +1,98 @@
+---
+title: 'HOHQMesh: An all Quadrilateral/Hexahedral Unstructured Mesh Generator for High Order Elements'
+tags:
+  - Mesh Generation
+  - Spectral Element Methods
+  - High Order
+  - Unstructured Quadrilateral and Hexahedral Meshes
+authors:
+  - name: David A. Kopriva
+    orcid: 0000-0002-8076-0856
+    affiliation: "1, 2"
+  - name: Andrew R. Winters
+    orcid: 0000-0002-5902-1522
+    corresponding: true
+    affiliation: 3
+  - name: Michael Schlottke-Lakemper
+    orcid: 0000-0002-3195-2536
+    affiliation: 4
+affiliations:
+ - name: Professor Emeritus, The Florida State University, Tallahassee, FL, USA
+   index: 1
+ - name: Adjunct Professor, San Diego State University, San Diego, CA, USA
+   index: 2
+ - name: Department of Mathematics; Applied Mathematics, Linköping University, Sweden
+   index: 3
+ - name: Professor, High-Performance Scientific Computing & Centre for Advanced Analytics and Predictive Sciences, University of Augsburg, Germany
+   index: 4
```

Comment:
@DavidAKopriva are you sure about the need for the position? At least in the docs they just use the regular affiliation, e.g., [here](https://joss.readthedocs.io/en/latest/paper.html#affiliations)

Reply:
It ought it was strange, but that's what they did on the example document. I don't see it anywhere else, so I will remove those.